Trying to compile a project created with `yarn create...` fails due to variants of `:react-native-camera`

I'm using Ubuntu 24.10 running on an Intel Core i5-8265U, 16GB in RAM and 100G of SDD available.

* First I run `sudo apt dist-upgrade`
* I followed the instructions of https://docs.divvi.xyz/mobile-framework/divvi-mobile/getting-started
  * I started with `yarn create expo --template https://github.com/divvi-xyz/divvi-app-starter my-app`
  * I followed "Expo's interactive setup guide" chossing "Android Device", "Development build", disabled "Build with Expo Application Services (EAS)"  and Linux.
    * I followed the "instructions from the Watchman documentation" <https://facebook.github.io/watchman/docs/install#linux> but since I couldn't find .deb binaries for the latest release I followed instructions to use homebrew:
      * I run the script as shown at <https://brew.sh/> and did the instruction presented to add to `.bashrc`
        ```sh
        $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
        $ echo >> /home/codespace/.bashrc
        $ echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/codespace/.bashrc
        $ eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
        $ sudo apt-get install build-essential
        $ brew update
        $ brew install gcc
        $ brew install watchman
        $ watchman version
        version: 2025.02.17.00
        ```
    * To install OpenJDK from Ubuntu packages I run
      ```
      $ sudo apt install openjdk-23-jdk
      $ javac --version
      javac 23.0.2
      ```
    * To install Android Studio I downloaded the most recent binaries for Linux from <https://developer.android.com/studio?hl=es-419> unzipped them and moved them to `/usr/local/`
      ```sh
      $ tar xvfz android-studio-2024.2.2.13-linux.tar.gz
      $ sudo mv android-studio /usr/local/
      $ /usr/local/android-studio/bin/studio
      ```
      After it completed installation I added to `~/.bashrc`:
      ```sh
      export ANDROID_HOME=/home/vtamara/Android/Sdk
      export PATH=$PATH:$ANDROID_HOME/emulator
      export PATH=$PATH:$ANDROID_HOME/platform-tools
      ```
      And run `source ~/.bashrc`
    * To activate KVM acceleration I followed <https://help.ubuntu.com/community/KVM/Installation>
      ```sh    
      $ sudo apt install cpu-checker
      $ kvm-ok
      INFO: /dev/kvm exists
      KVM acceleration can be used
      $ sudo apt-get install qemu-kvm libvirt-daemon-system libvirt-clients bridge-utils
      $ sudo adduser `id -un` libvirt
      $ sudo adduser `id -un` kvm
      $ sudo reboot
      ```
  * From the app directory I run `yarn android`, it launched the Android emulator.  However the compilation of the app failed with:

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all dependencies for configuration ':app:debugCompileClasspath'.
   > Could not resolve project :react-native-camera.
     Required by:
         project :app
      > The consumer was configured to find a library for use during compile-time, preferably optimized for Android, as well as attribute 'com.android.build.api.attributes.AgpVersionAttr' with value '8.6.0', attribute 'com.android.build.api.attributes.BuildTypeAttr' with value 'debug', attribute 'org.jetbrains.kotlin.platform.type' with value 'androidJvm'. However we cannot choose between the following variants of project :react-native-camera:
          - generalDebugApiElements
          - mlkitDebugApiElements
        All of them match the consumer attributes:
          - Variant 'generalDebugApiElements' capability 'BancoCCOP:react-native-camera:unspecified' declares a component for use during compile-time, as well as attribute 'com.android.build.api.attributes.AgpVersionAttr' with value '8.6.0', attribute 'com.android.build.api.attributes.BuildTypeAttr' with value 'debug':
              - Unmatched attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about its target Java environment (preferred optimized for Android)
                  - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'androidJvm')
                  - Provides attribute 'com.android.build.api.attributes.ProductFlavor:react-native-camera' with value 'general' but the consumer didn't ask for it
                  - Provides attribute 'com.android.build.gradle.internal.attributes.VariantAttr' with value 'generalDebug' but the consumer didn't ask for it
                  - Provides attribute 'react-native-camera' with value 'general' but the consumer didn't ask for it
          - Variant 'mlkitDebugApiElements' capability 'BancoCCOP:react-native-camera:unspecified' declares a component for use during compile-time, as well as attribute 'com.android.build.api.attributes.AgpVersionAttr' with value '8.6.0', attribute 'com.android.build.api.attributes.BuildTypeAttr' with value 'debug':
              - Unmatched attributes:
                  - Doesn't say anything about its component category (required a library)
                  - Doesn't say anything about its target Java environment (preferred optimized for Android)
                  - Doesn't say anything about org.jetbrains.kotlin.platform.type (required 'androidJvm')
                  - Provides attribute 'com.android.build.api.attributes.ProductFlavor:react-native-camera' with value 'mlkit' but the consumer didn't ask for it
                  - Provides attribute 'com.android.build.gradle.internal.attributes.VariantAttr' with value 'mlkitDebug' but the consumer didn't ask for it
                  - Provides attribute 'react-native-camera' with value 'mlkit' but the consumer didn't ask for it

* Try:
> Ambiguity errors are explained in more detail at https://docs.gradle.org/8.10.2/userguide/variant_model.html#sub:variant-ambiguity.
> Review the variant matching algorithm at https://docs.gradle.org/8.10.2/userguide/variant_attributes.html#sec:abm_algorithm.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
```

--- a/app.json
+++ b/app.json
@@ -80,11 +80,9 @@
     ],
     "extra": {
       "eas": {
-        // Replace with your own project ID
         "projectId": "1c1a9df8-1e2d-45ef-8667-90a2b9e0d5d3"
       }
     },
-    // Replace with your own owner
     "owner": "divvi"
   }
 }

